.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
   - SYMFONY_VERSION=2.5.*
 
 matrix:
+  allow_failures:
+    - php: hhvm
```

Comment:
we should just add it as normal test and only do this when there is a problem with HHVM support.